cli/src/diff_util.rs

```diff
@@ -572,20 +579,28 @@ pub fn show_color_words_diff(
     let mut diff_stream = materialized_diff_stream(store, tree_diff);
     async {
         while let Some(MaterializedTreeDiffEntry {
-            source: _,
-            target: path,
+            source: left_path,
+            target: right_path,
             value: diff,
         }) = diff_stream.next().await
         {
-            let ui_path = path_converter.format_file_path(&path);
+            let left_ui_path = path_converter.format_file_path(&left_path);
+            let right_ui_path = path_converter.format_file_path(&right_path);
             let (left_value, right_value) = diff?;
 
             match (&left_value, &right_value) {
-                (_, MaterializedTreeValue::AccessDenied(source))
-                | (MaterializedTreeValue::AccessDenied(source), _) => {
+                (MaterializedTreeValue::AccessDenied(source), _) => {
                     write!(
                         formatter.labeled("access-denied"),
-                        "Access denied to {ui_path}:"
+                        "Access denied to {left_ui_path}:"
+                    )?;
+                    writeln!(formatter, " {source}")?;
+                    continue;
+                }
+                (_, MaterializedTreeValue::AccessDenied(source)) => {
+                    write!(
+                        formatter.labeled("access-denied"),
+                        "Access denied to {right_ui_path}:"
```

Comment:
Wasn't the point of pass an empty `CopyRecordMap` on line 308 so we don't have to worry about the `source` field here? I.e., can we undo the changes to this function?

Reply:
oh, I forgot that I had implemented this one.  I fixed line 308 and updated the CHANGELOG.